src/views/templates/list/VirtualMachineTemplatesColumns.tsx

```diff
@@ -0,0 +1,28 @@
+import { TFunction } from 'i18next';
+
+import { K8sResourceCommon, TableColumn } from '@openshift-console/dynamic-plugin-sdk';
+
+const VirtualMachineTemplatesColumns: (t: TFunction) => TableColumn<K8sResourceCommon>[] = (t) => [
+  {
+    title: t('Name'),
+    id: 'name',
+  },
+  {
+    title: t('Namespace'),
+    id: 'namespace',
+  },
+  {
+    title: t('Template provider'),
+    id: 'provider',
+  },
+  {
+    title: t('Source'),
+    id: 'source',
+  },
+  {
+    title: t('Actions'),
+    id: 'actions',
+  },
```

Comment:
```suggestion
   {
    title: '',
    id: '',
    props: { className: 'dropdown-kebab-pf pf-c-table__action'}, // once we have Kebab{ className: Kebab.columnClass }
  },
```
actions column doesn't have a title

Reply:
cc:// @vojtechszocs about the Kebab ^^